README.md

```diff
@@ -1,51 +1,58 @@
-# Swiftype WordPress Plugin
+<p align="center"><img src="https://github.com/swiftype/swiftype-wordpress-php/blob/master/logo-site-search.png?raw=true" alt="Elastic Site Search Logo"></p>
 
-The [Swiftype WordPress Plugin](https://swiftype.com/wordpress) replaces WordPress's default search with a better search engine that is fully customizable via the Swiftype dashboard.
+> The [Site Search WordPress Plugin](https://swiftype.com/wordpress) replaces WordPress's default search with a better search engine that is fully customizable via the Site Search dashboard.
 
-## Installation
+## Contents
 
-1. Go to [http://swiftype.com](http://swiftype.com) and sign up for a free Swiftype account. (Be sure to validate your account via the confirmation email we send.)
-2. After logging in to Swiftype, go to the Account Settings screen and get your API key.
-3. Install the Swiftype Search Wordpress plugin in your Wordpress dashboard.
-4. Activate the plugin through the 'Plugins' menu in WordPress.
-5. Go to the Swiftype Search plugin page and enter your Swiftype API key on the first screen.
-6. Name your search engine, following the instructions on the screen.
-7. Build your search index by clicking the "Synchronize with Swiftype" button.
-8. See the Demo video for additional details, or email support@swiftype.com if you are having trouble.
+- [Getting started](#getting-started-)
+- [Usage](#usage)
+- [FAQ](#faq-)
+- [Contribute](#contribute-)
+- [License](#license-)
+
+***
+
+## Getting started 🐣
+
+It is recommended that you install the plugin from the Wordpress Plugin Management screen of your Wordpress install (**Plugins -> Add New**);
+
+Once the plugin is installed and enabled, you will see a new **Site Search** entry in the admin menu. Go to this entry to configure the Site Search plugins :
+
+  1. Enter your Swiftype API key on the first screen.
+  3. Name your search engine and optionnaly choose a language.
+  3. Build your search index by clicking the "Synchronize" button.
+
+**Note :**
+  * Using this plugin assumes that you have already created a Site Search account on https://swiftype.com/.
+  * If you already have an account, you will need your API Key in order to configure the plugin. You can find it in on top of the Site Search [Account Settings](https://app.swiftype.com/settings/account) screen on the Site Search site.
```

Comment:
I think all external links should open in a new tab, so that it is clearer to a user when they are in Wordpress and when they are on swiftype.com.

Reply:
Would be great but I was not able to find a way to do it using GitHub flavored Markdown.
Maybe our Chief Fanciness Officer @goodroot know a way to get this done.

Reply:
Sorry, I put this comment in the wrong place. Disregard.